sonar/common/jsonschemas/type-v1.0.0.json

```diff
@@ -2,88 +2,276 @@
+        "value": "coar:c_2cd9",
+        "level": 1
+      },
+      {
+        "label": "document_type_coar:c_2fe3",
+        "value": "coar:c_2fe3",
+        "level": 1
+      },
       {
         "label": "document_type_coar:c_816b",
         "value": "coar:c_816b"
       },
       {
-        "label": "document_type_coar:c_6501",
-        "value": "coar:c_6501"
+        "label": "document_type_coar:c_93fc",
+        "value": "coar:c_93fc"
       },
       {
-        "label": "document_type_coar:c_3248",
-        "value": "coar:c_3248"
+        "label": "document_type_coar:c_18ww",
+        "value": "coar:c_18ww",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_5794",
-        "value": "coar:c_5794"
+        "label": "document_type_coar:c_18wz",
+        "value": "coar:c_18wz",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_2f33",
-        "value": "coar:c_2f33"
+        "label": "document_type_coar:c_18wq",
+        "value": "coar:c_18wq",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_bdcc",
-        "value": "coar:c_bdcc"
+        "label": "document_type_coar:c_186u",
+        "value": "coar:c_186u",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_7a1f",
-        "value": "coar:c_7a1f"
+        "label": "document_type_coar:c_18op",
+        "value": "coar:c_18op",
+        "level": 1
+      },
+      {
+        "label": "document_type_coar:c_ba1f",
+        "value": "coar:c_ba1f",
+        "level": 1
+      },
+      {
+        "label": "document_type_coar:c_18hj",
+        "value": "coar:c_18hj",
+        "level": 1
+      },
+      {
+        "label": "document_type_coar:c_18ws",
+        "value": "coar:c_18ws",
+        "level": 1
+      },
+      {
+        "label": "document_type_coar:c_18gh",
+        "value": "coar:c_18gh",
+        "level": 1
       },
       {
         "label": "document_type_coar:c_46ec",
         "value": "coar:c_46ec"
       },
       {
-        "label": "document_type_coar:c_db06",
-        "value": "coar:c_db06"
+        "label": "document_type_coar:c_7a1f",
+        "value": "coar:c_7a1f",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_12cc",
-        "value": "coar:c_12cc"
+        "label": "document_type_coar:c_db06",
+        "value": "coar:c_db06",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_18ws",
-        "value": "coar:c_18ws"
+        "label": "document_type_coar:c_bdcc",
+        "value": "coar:c_bdcc",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_2fe3",
-        "value": "coar:c_2fe3"
+        "label": "document_type_habilitation_thesis",
+        "value": "habilitation_thesis",
+        "level": 1
       },
       {
-        "label": "document_type_coar:c_0640",
-        "value": "coar:c_0640"
+        "label": "document_type_advanced_studies_thesis",
```

Comment:
For instance, here I would have put "Advanced studies thesis". In my understanding, that is why we use label, to provide a human translation of a code.